intro/setup/index.html

```diff
@@ -347,16 +384,19 @@ <h2><a id="final-result" class="js"></a>Final Result</h2>
 WIN_SIZE = (420, 250)
 screen = pygame.display.set_mode(WIN_SIZE, flags=pygame.SCALED)
 
+is_running = True
 while True:
```

Comment:
Instead of `while True` it should be `while is_running`, otherwise `is_running` isn't doing anything